extensions/vscode/src/utils/config.ts

```diff
@@ -0,0 +1,47 @@
+import * as fs from 'fs';
+import * as path from 'path';
```

Comment:
Nitpicky but just to match other parts of the codebase and use the default export:

```suggestion
import fs from "fs";
import path from "path";
```